recipes/python/patches/ctypes-find-library.patch

```diff
@@ -0,0 +1,21 @@
+--- Python-2.7.2.orig/Lib/ctypes/util.py	2011-06-11 16:46:24.000000000 +0100
++++ Python-2.7.2/Lib/ctypes/util.py	2015-03-26 13:45:33.322172603 +0000
+@@ -83,7 +83,17 @@
+             except ValueError:
+                 continue
+         return None
+-
++elif os.name == "posix" and sys.platform == "linux3":
```

Comment:
@kived I have created a new patch removing the system checks and just leaving the find_library function as suggested.

Can some people test and let us know if it works and perhaps help get it merged :)